packages/node_modules/@ciscospark/react-redux-spark/src/spark.js

```diff
@@ -10,62 +10,70 @@ import Spark from '@ciscospark/spark-core';
 import LocalStorageStoreAdapter from '@ciscospark/storage-adapter-local-storage';
 
 export default function createSpark(accessToken) {
-  const credentials = JSON.parse(localStorage.getItem(`credentials`) || false);
-  return new Spark({
-    credentials: {
-      authorization: {
-        access_token: accessToken || credentials.access_token
-      }
-    },
-    config: {
-      logger: {
-        level: process.env.NODE_ENV === `production` ? undefined : `trace`
+  // Check if we already have a spark instance in memory
+  let ciscosparkInstance = window.ciscosparkInstance;
```

Comment:
I originally wanted to but didn't want to chance the instance getting overwritten by someone attaching the SDK there.